config/routes.rb

```diff
@@ -211,6 +212,18 @@
     get '/sign_up' => 'users#new', as: 'sign_up' if Clearance.configuration.allow_sign_up?
   end
 
+  ################################################################################
+  # UI API
```

Comment:
@jenshenny The distinction is made as "UI API" here because these are JSON routes that are used to talk to the WebAuthn API (https://www.w3.org/TR/webauthn-2/#sctn-createCredential). They are not ones that the user interacts with directly

Reply:
I see, that makes sense now. If these routes are used to interact with the WebAuthn API, could we make it more explicit by naming it `UI WebAuthn API` or similiar?